src/Domain/CoordinateMaps/KerrHorizon.hpp

```diff
@@ -0,0 +1,68 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+
+namespace domain::CoordinateMaps {
+
+/*!
+ * Distorts cartesian coordinates \f$X^i\f$ such that a coordinate sphere
```

Comment:
could you add a `\brief`. Maybe something like the first sentence up to the `, defined as the largest...`?